apps/web/src/features/spaces/components/SpacesDashboardWidget/index.tsx

```diff
@@ -25,49 +19,37 @@ const SpacesDashboardWidget = () => {
   if (widgetHidden) return null
 
   return (
-    <>
-      <Stack direction="row" flexWrap="wrap" gap={2} p={3} sx={gradientBg} position="relative">
-        <Box sx={{ position: 'absolute', right: 24, top: 16 }}>
-          <Track {...SPACE_EVENTS.HIDE_DASHBOARD_WIDGET}>
-            <IconButton aria-label="close" onClick={onHide} size="small">
-              <CloseIcon fontSize="medium" />
-            </IconButton>
-          </Track>
-        </Box>
-
-        <Box flex={1} minWidth="60%">
-          <Chip label="Beta" sx={{ backgroundColor: '#12FF80', borderRadius: '4px' }} size="small" />
-
-          <Typography variant="h6" fontWeight="700" mb={2} mt={1}>
-            Spaces are here!
-          </Typography>
-
-          <Typography variant="body2">
-            Organize your Safe Accounts, all in one place. Collaborate efficiently with your team members and simplify
-            treasury management.
-            <br />
-            Available now in beta.
-          </Typography>
-        </Box>
-
-        <Stack direction="row" gap={2} alignItems="flex-end">
-          <Track {...SPACE_EVENTS.INFO_MODAL} label={SPACE_LABELS.safe_dashboard_banner}>
-            <Button variant="outlined" onClick={() => setIsInfoOpen(true)}>
-              Learn more
+    <Stack
+      direction={{ xs: 'column', sm: 'row' }}
+      flexWrap="wrap"
+      gap={2}
+      p={2}
+      sx={{ backgroundColor: 'secondary.light' }}
+      position="relative"
+    >
+      <Box flex={1} minWidth="60%">
+        <Typography>
+          🎉 <b>New! Improved Spaces:</b> All your Safe Accounts, finally organized. Streamlined for teams and solo
```

Comment:
Is this meant to be in this release?

Reply:
I originally pushed it into the address book epic but since we didn't include it in the release I had to cherry-pick it from there.